### PR TITLE
Project Documentation (mihai.tigaeru)

### PR DESCRIPTION
**Pull request by septimium**
### Pull Request Overview

This pull request adds the index.md file with required assets to the project folder.

### TODO or Help Wanted

Waiting for approval.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Mihai-Septimiu Tigaeru <septimiutg@gmail.com>

### CODE REVIEW

**Comment by septimium** [comment]
Is there anything I should modify besides the Architecture?

File: docs/project/2024/mihai.tigaeru/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Pico Snake Game
+A game in rust that resembles the classic Snake Game from the Nokia 5110 phones era.
+
+:::info 
+
+**Author**: Mihai-Septimiu Tigaeru \
+**GitHub Project Link**: (https://github.com/UPB-FILS-MA/project-septimium)
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.tigaeru/index.md:7]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-septimium
```

File: docs/project/2024/mihai.tigaeru/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+TBD
+
+### Bill of Materials
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico H](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [40 RON](https://www.bitmi.ro/placi-de-dezvoltare/placa-de-dezvoltare-raspberry-pi-pico-h-rp2040-264kb-ram-10848.html) |
+| 2-axis (XY) Joystick Module | Game Interactions | [6 RON](https://www.bitmi.ro/module-electronice/modul-joystick-cu-2-axe-x-y-10454.html) |
+| Breadboard 830 | Hardware Pins Connections | [9 RON](https://www.bitmi.ro/componente-electronice/breadboard-830-puncte-mb-102-10500.html) |
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.tigaeru/index.md:56]: Please add links to the devices, not just the store where you buy them.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by septimium** [comment]
Is it ok now?

**Review by alexandruradovici — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Yes

**Review by DanutAldea — APPROVED** [review]

**Pull request merged**